### PR TITLE
fixed audio bugs

### PR DESCRIPTION
**Pull request by devopsean**

### CODE REVIEW

**Comment by devopsean** [comment]
Removed second button

File: lib/widgets/menu.dart [hunk 1]
```diff
@@ -23,7 +29,11 @@ class Menu extends StatelessWidget {
             mainAxisAlignment: MainAxisAlignment.spaceAround,
             crossAxisAlignment: CrossAxisAlignment.start,
             children: <Widget>[
-              Sound(toggleMusic, isPlaying),
```
Comment (Kymoraa) [inline lib/widgets/menu.dart:26]: I'm not sure why we are moving from toggleMusic and isPlaying to toggleSound and soundOn

File: lib/board.dart [hunk 2]
```diff
@@ -195,21 +194,45 @@ class _BoardState extends State<Board> with WidgetsBindingObserver {
...
-      );
+      _assetsAudioPlayer.playOrPause();
+      setState(() {
+        isPlaying = true;
+      });
+    }
+  }
 
-      isPlaying = true;
+  void toggleSound() {
```
Comment (Kymoraa) [inline lib/board.dart:225]: Please explain why we have this function?
If we have just the one button for music on and off

File: lib/board.dart [hunk 3]
```diff
@@ -24,7 +24,8 @@ class _BoardState extends State<Board> with WidgetsBindingObserver {
   bool isStarted = false;
   bool isPlaying = false;
   final firstLoadedKey = "true";
-
+  bool soundOn = true;
+  bool musicStarted = false;
```
Comment (Kymoraa) [inline lib/board.dart:28]: Also, clarify why we need the two isPlaying vs musicStarted

**Comment by devopsean** [comment]
Hi Jackie,

Those were part of the former functionality for two buttons. I commented
out the second button afterwards. I could remove the unneeded codes.
The music doesn't start from beginning anymore,
Sean

**Pull request closed without merging**